resources/dicts/patrols/new_cat.json

```diff
@@ -458,8 +458,8 @@
     "fail_trait": ["vengeful", "bloodthirsty"],
     "min_cats": 2,
     "max_cats": 6,
-    "antagonize_text": null,
-    "antagonize_fail_text": null,
+    "antagonize_text": "The patrol follows the sound, intent on finding the intruder. A kittypet! No wonder they were so loud. they scare them off easily.",
+    "antagonize_fail_text": "The patrol follows the noise and comes upon a kittypet that's sniffing around. The kittypet quickly begins a story about how they've been training for <i>days</i> now and they're finally ready to join c_n. p_l laughs and gets clawed.",
```

Comment:
the "gets clawed" part of this could use a bit more context, I think.  it doesn't really click immediately that the kittypet has clawed them